assets/scss/collections.scss

```diff
@@ -0,0 +1,26 @@
+---
```

Comment:
main thing i want to be sure we don't break before merging as-is:

the collections page has custom styles